keyboards/Emi20/Emi20.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
```

Comment:
This file needs to be renamed to lowercase.

Reply:
I will get on the lowercase matter, somehow my pc doesn't want to push anything to github so I needed to update through the desktop app. Will try to edit the names now.

Reply:
Well this sucks, while trying to rename the main Emi20 folder accidentally moved config.h 🤦 Let's see how I'm fixing this

Reply:
`git mv keyboards/Emi20 keyboards/tmp`, then `git mv keyboards/tmp keyboards/emi20` should do the trick.

Reply:
I'll try that again, if it still doesn't work I will see what the feck is wrong with my git xD

Oh and wouldn't this result in two emi20 folders as I made that mistake before with the config.h file? I could use `-f` to force it but not sure what would happen then ¯\_(ツ)_/¯

Reply:
No, because you are first renaming the folder to `tmp`, then back to the original name but lowercase (this is because sometimes Git doesn't see case-only filename changes - it depends how your `.gitconfig` is set up)

Reply:
I get that but as I made a mistake previously with the config.h file there is also a emi20 all lowercase folder already present but I guess the files will then just be moved to that folder instead of renaming the tmp folder to emi20?

In any case I'll try to do that.

Reply:
Ah, well in that case you could probably move each file individually.

Reply:
Ah ok, I am still trying to get my local copy of the repository synced back up. No clue how it got desynced but it's a mess xD

If I can fix it I will move the files, after that no clue how we need to fix the config.h and rules.mk file in the default keymap folder. and again thank you so much for all your help!! qmk and git in general 😃